data/pokemongo.ts

```diff
@@ -0,0 +1,961 @@
+/**
```

Comment:
Normally we put encounter data in a `learnsets.ts` file — is there a reason you chose to split off Pokémon GO encounters into their own file?

Reply:
I prefer the data being split from learnsets, the learnsets file is already a massive file on its own and is a pain to edit, so id prefer if we didnt have even more encounter legality data in there